Reference/Cache/index.md

```diff
@@ -61,12 +62,14 @@ When an `ICacheRefresher` is executed via the `DistributedCache.Instance` a noti
 
 ## Events handling to refresh cache
 
-TODO: fill in these docs(https://github.com/umbraco/UmbracoDocs/issues/208)
+To use the extensions add a using to `Umbraco.Web.Cache`;  You can then call them on the DistributedCache.Instance object.
 
 ## IServerMessenger
 
-TODO: fill in these docs(https://github.com/umbraco/UmbracoDocs/issues/208)
+The server messenger broadcasts distributed cache notifications to all servers of a load balanced environment.
```

Comment:
'distributed cache notifications'  to each server in the load balanced environment.